docs/RELEASE.md

```diff
@@ -1,12 +1,16 @@
 # Release process
 
+* Ensure you have a backup of all working files and then remove files not tracked by git
+  `git clean -xdf`. **NOTE**: this will delete all files in the tuf tree that aren't
```

Comment:
Does this mean that if I am to release something, this will have to delete all my venvs in the current directory if I don't back them up?

Reply:
Yes, it does. The theory is that it's better to ensure the directory is clean and matches the git tree, so that we don't accidentally end up packaging things we don't intend to ship.

Reply:
Yes, it makes sense.
Just wanted to make sure I understand it right.

Reply:
@MVrachev, you could also keep your venv's in an [`ENV_DIR`](https://docs.python.org/3/library/venv.html) that's not under cwd. :)

Reply:
@MVrachev the alternative I've used is that I never do releases from my development directory: I make a new git clone for every release. This ensures two things
1. all files required for the release actually are in git 
1. no files from outside of git end up in the release